core/indexer/elasticsearch_test.go

```diff
@@ -269,7 +241,8 @@ func TestElasticIndexer_SaveValidatorsPubKeys(t *testing.T) {
 
 	keys := [][]byte{[]byte("key")}
 	valPubKey[0] = keys
-	ei.SaveValidatorsPubKeys(valPubKey)
+	epoch := uint32(0)
```

Comment:
add a test for epoch start notifier and you can also check if the ES is updated when a new epoch starts

Reply:
done.